velox/type/Type.cpp

```diff
@@ -488,6 +488,14 @@ bool RowType::equals(const Type& other) const {
   return true;
 }
 
+size_t RowType::hashKind() const {
```

Comment:
This seems like unrelated change? Would it be possible to extract it into a separate PR and add a test?